requirements.txt

```diff
@@ -30,5 +30,5 @@ six==1.11.0
 SQLAlchemy==1.3.10
 typing==3.6.6
 urllib3==1.24.2
-Werkzeug==0.15.3
+Werkzeug==0.15.5
```

Comment:
Make sure you rebase from master so that `Updated version of Graphene and Werkzeug` <- this commit isn't a part of this PR